h5p-bildetema/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -22,7 +25,12 @@ export const Breadcrumbs: React.FC<BreadcrumbsProps> = ({
   breadCrumbs,
   currentLanguageCode,
 }) => {
-  const topicLabel = useL10n("breadcrumbsTopic");
+  const { translations } = useDBContext() || {};
+  const labelFromDb = getLabelFromTranslationRecord(
+    translations?.[currentLanguageCode as LanguageCode],
```

Comment:
We should prefer setting `currentLanguageCode`'s type to `LanguageCode` in the props to avoid `as` as much as possible.